bazel/BUILD

```diff
@@ -29,3 +29,16 @@ cc_grpc_library(
     proto_only = True,
     deps = [],
 )
+
```

Comment:
looks like this is not being used anywhere? Let's only add once we actually have a use for this.

Reply:
removed